src/components/ViewDeck/ViewDeck.js

```diff
@@ -100,36 +107,36 @@ class ViewDeck extends Component {
     //.reduce
 
     cards.forEach((card) => {
-      if (card.colors.length === 0 && card.cmc > 0) {
-        console.log("colorless card with cmc of", card.cmc);
-        devotion.Gray += card.cmc;
+      if (card.jsonData.colors.length === 0 && card.jsonData.cmc > 0) {
+        console.log("colorless card with cmc of", card.jsonData.cmc);
+        devotion.Gray += card.jsonData.cmc;
       }
 
-      for (let index = 0; index < card.mana_cost.length; index++) {
-        const element = card.mana_cost[index];
+      for (let index = 0; index < card.jsonData.mana_cost.length; index++) {
+        const element = card.jsonData.mana_cost[index];
         if (element.includes("{") || element.includes("}")) {
           //console.log('this includes a { or }');
         } else {
           console.log("this is", element);
           if (element === "W") {
             //console.log('white');
-            devotion.White += 1;
+            devotion.White += (1 * card.quantity);
```

Comment:
This should definitely work for the quantity. Once this file is switched to functional, it would be worth going over the logic in your methods. Switch to .reduce or .map where possible.